### DISCUSSION TITLE
"Please check your internet connection" error

### DISCUSSION THREAD

**Discussion by hitarthbhatt2311** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#531] [General]
Even after connecting the computer to a network, the assistant is rendered useless and shows the error, "Getting metadata from plugin failed with error: invalid grant" Please help, I'm stuck with Cortana. 😭

**Comment by Melvin-Abraham** [comment]
You can refer to [this section of FAQ](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)#how-do-i-fix-invalid_grant-error) for the time being. A better way to handle this error will be available in the next update.
